pkg/dataplane/util/ansible_execution.go

```diff
@@ -178,7 +178,7 @@ func (a *EEJob) BuildAeeJobSpec(
 	service *dataplanev1.OpenStackDataPlaneService,
 	nodeSet client.Object,
 ) {
-	const jobRestartPolicy string = "OnFailure"
+	const jobRestartPolicy string = "Never"
```

Comment:
could use the corev1.RestartPolicyNever const instead of using a custom const?

Reply:
sure, done! There is quite a bit of refactoring still to be done...